.github/workflows/build.yml

```diff
@@ -11,6 +11,16 @@ jobs:
     name: Build binaries
     runs-on: ubuntu-latest
     environment: "Build, sign, release binaries"
+    strategy:
+      matrix:
+        include:
+          - {os: linux, arch: amd64}
+          - {os: linux, arch: arm, arm: 6}
+          - {os: linux, arch: arm64}
+          - {os: darwin, arch: amd64}
+          - {os: darwin, arch: arm64}
+          - {os: windows, arch: amd64}
+          - {os: freebsd, arch: amd64}
```

Comment:
As a matter of style, let's keep calling these GOOS/GOARCH/GOARM.

Reply:
Resolved with https://github.com/FiloSottile/age/pull/456/commits/d9f5a6c7d08cb48a5ebf53da3c11b44ea08085df 🙃